test-cases/performance/perf-regression-nosqlbench-throughput-mixed-50r-50w.yaml

```diff
@@ -0,0 +1,7 @@
+stress_cmd: "nosqlbench cql-tabular2 rampup-cycles=1M main-cycles=100M write_ratio=5 read_ratio=5 threads=20x async=500 --progress console:300s rf=3 partsize=5000 driver=cql-scylla -v"
+n_db_nodes: 3
+n_loaders: 2
```

Comment:
Did you decide to go with 2 loaders?
Do you have results with 1 vs 2 or is it still in your testing?

Reply:
I went with your suggestion and tried 2 loaders to saturate the cluster. I'm testing what would be the best set of params to get to that saturation with this setup.